internal/btf/btf_test.go

```diff
@@ -110,6 +107,20 @@ func TestParseVmlinux(t *testing.T) {
 	}
 }
 
+func BenchmarkParseVmlinux(b *testing.B) {
+	for n := 0; n < b.N; n++ {
+		spec, err := parseVmLinux(b)
+		if err != nil {
+			b.Fatal(err)
+		}
+
+		// Access spec to prevent it from being optimized away.
```

Comment:
Does that actually happen?

Reply:
Thanks, was a leftover from a troubleshooting session. (forgot `b.N` loop)